### PR TITLE
Add german language

### PR DESCRIPTION
**Pull request by niklas-la**

### CODE REVIEW

File: resources/de.ini [hunk 1]
```diff
@@ -0,0 +1,45 @@
+; Updated time : 26th 07 2017
+; general
+divider = "========================================"
+spacer = " -=+=- {%0} -=+=- "
+error = "Da ist etwas schief gelaufen"
+noperm = "Sie haben nicht die erforderlichen Rechte um diesen Befehl ausführen zu können"
+runingame = "Bitte führen sie diesen Befehl im Spiel aus"
+commands = "Kommandos"
```
Comment (inxomnyaa) [inline resources/de.ini]: Would name it "Befehle"

File: resources/de.ini [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+ui.brush.options.blocks = "Blöcke"
+ui.brush.options.blocks.placeholder = "Beispiel: 1:1,2,tnt,log:12"
+ui.brush.options.diameter = "Durchmesser"
+ui.brush.options.width = "Breite"
+ui.brush.options.height = "Höhe"
+ui.brush.options.depth = "Tiefe"
+ui.brush.options.flags = "Option hinzufügen?"
+; ui flags
+ui.flags.keepexistingblocks = "Bestehende Blöcke beibehalten"
+ui.flags.keepair = "Halte Luft"
```
Comment (inxomnyaa) [inline resources/de.ini]: Would name it "Luft beibehalten"

File: resources/de.ini [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+ui.flags.hollow = "Hohlraum"
+ui.flags.natural = "Natürlich"
+; ui brush sphere
+; ui brush cylinder
+; ui brush cuboid
+; ui brush clipboard
+; ui flood
+ui.flood.title = "Flügssigkeitsmenü"
```
Comment (inxomnyaa) [inline resources/de.ini]: Would name "Fluss-Füller Menü"

**Comment by inxomnyaa** [comment]
Please name the file "deu.ini" and keep "eng.ini" by its old name

Thanks for the PR, could you apply the commented changes?

**Comment by niklas-la** [comment]
Commented changes are applied 😉

**Comment by Aericio** [comment]
Looks good, thank you!

**Pull request merged**